src/constraints/load_balance.jl

```diff
@@ -66,8 +66,19 @@ function add_elec_load_balance_constraints(m, p; _n="")
 		sum( m[Symbol("dvDischargeFromStorage"*_n)][b,ts] for b in p.s.storage.types.elec )  ==
         sum( sum(m[Symbol("dvProductionToStorage"*_n)][b, t, ts] for b in p.s.storage.types.elec) + 
         m[Symbol("dvCurtail"*_n)][t, ts] for t in p.techs.elec) +
-        p.s.electric_load.critical_loads_kw[ts]
+        (p.s.electric_load.critical_loads_kw[ts] * m[Symbol("dvOffgridLoadServedFraction"*_n)][ts] )
 	)
+
+    ##Constraint : For off-grid scenarios, annual load served must be >= minimum percent specified  
+    if !p.s.settings.off_grid_flag # fix dvOffgridLoadServedFraction to 100% for "on-grid" analyses
+		for ts in p.time_steps_without_grid
+			fix(m[Symbol("dvOffgridLoadServedFraction"*_n)][ts], 1.0, force=true) ## TODO: Check if this messes with the dvUnservedLoad outage constraints
```

Comment:
@NLaws This step of fixing the "load served fraction" to 100% for on-grid runs is something we discussed before (you had suggested this as a way to trim down the code by not creating a separate load balance constraint for off-grid runs). However, I'm wondering if you think this will now be an issue with the new outage modeling capabilities (related to dvUnservedLoad)? I haven't done too much digging into this yet but wanted to flag it now in case you see any issues off the bat.

Reply:
The `dvUnservedLoad` variable is only created if `p.s.electric_utility.outage_durations` is non-empty. Can we also create `dvOffgridLoadServedFraction` conditionally?  No outages are modeled with `off_grid_flag=true` right?  We could provide a warning when a user provides outage information and `off_grid_flag=true` that the outage information is not used when `off_grid_flag=true`.